doc/users/next_whats_new/sides_violinplot.rst

```diff
@@ -1,4 +1,4 @@
 Add option to plot only one half of violin plot
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Setting the parameter `side` to 'left' or 'right' allows to only plot one half of the violin plot.
+Setting the parameter `side` to 'low' or 'high' allows to only plot one half of the violin plot.
```

Comment:
```suggestion
Setting the parameter *side* to 'low' or 'high' allows to only plot one half of the violin plot.
```

"code" style in rst requires double backticks (not single, like markdown), but the standard for parameter names is `*` (emphasis)

(ref https://matplotlib.org/devdocs/devel/document.html#function-arguments)

This is why the doc build is failing.

Reply:
Ahhhh thank you!